ml-agents/mlagents/trainers/trainer_util.py

```diff
@@ -39,6 +39,10 @@ def __init__(
         self.ghost_controller = GhostController()
 
     def generate(self, brain_name: str) -> Trainer:
+        if brain_name not in self.trainer_config.keys():
+            logger.warning(
+                f"Behavior name {brain_name} does not match any behaviors specified in the trainer configuration file."
```

Comment:
```suggestion
                f"Behavior name {brain_name} does not match any behaviors specified in the trainer configuration file: {sorted(self.trainer_config.keys())}"
```
(that might not pass precommit)